ArduPlane/precision_landing.cpp

```diff
@@ -0,0 +1,31 @@
+//
+// functions to support precision landing
+//
+
+#include "Plane.h"
+
+#if PRECISION_LANDING == ENABLED
+void Plane::init_precland()
+{
+    plane.precland.init(400);

```

Comment:
Constants are bad.  Find the method on AP::scheduler() :-)

Reply:
Is this wrong in copter as well, because it is the same code? I am a little confused by that. I did add a scheduler method in ArduPlane.cpp but the Precland library init function takes an integer parameter; frequency at which to update. Are you saying I should use a #define instead?

Reply:
Done